pyanaconda/payload/live/payload_liveimg.py

```diff
@@ -126,47 +123,14 @@ def pre_install(self):
         task.progress_changed_signal.connect(self._progress_cb)
         task.run()
 
-        # If this looks like a tarfile, skip trying to mount it
-        if is_tar(self.data.liveimg.url):
-            return
-
-        # Work around inability to move shared filesystems.
-        # Also, do not share the image mounts with /run bind-mounted to physical
-        # target root during storage.mount_filesystems.
-        rc = util.execWithRedirect("mount",
-                                   ["--make-rprivate", "/"])
-        if rc != 0:
-            log.error("mount error (%s) making mount of '/' rprivate", rc)
-            raise PayloadInstallError("mount error %s" % rc)
-
-        # Mount the image and check to see if it is a LiveOS/*.img
-        # style squashfs image. If so, move it to IMAGE_DIR and mount the real
-        # root image on INSTALL_TREE
-        rc = payload_utils.mount(self.image_path, INSTALL_TREE, fstype="auto", options="ro")
-        if rc != 0:
-            log.error("mount error (%s) with %s", rc, self.image_path)
-            raise PayloadInstallError("mount error %s" % rc)
-
-        # Nothing more to mount
-        if not os.path.exists(INSTALL_TREE + "/LiveOS"):
-            return
-
-        # Mount the first .img in the directory on INSTALL_TREE
-        img_files = glob.glob(INSTALL_TREE + "/LiveOS/*.img")
-        if img_files:
-            # move the mount to IMAGE_DIR
-            os.makedirs(IMAGE_DIR, 0o755)
-            rc = util.execWithRedirect("mount",
-                                       ["--move", INSTALL_TREE, IMAGE_DIR])
```

Comment:
There is no indication why this code is not needed anymore. Can you please describe that in the commit message?

Reply:
The commit message was updated.